aries-site/src/pages/components/card.mdx

```diff
@@ -69,34 +147,132 @@ A card by nature has limited real-estate, so it should present summarized inform
+   - b. **Pretitle** (optional): Additional context for what this call-to-action is about (e.g., "Virtual Event").
+   - c. **Subtitle** (optional): Any concise, additional supportive text to help establish the card's identity and context.
+1. **Actions** (required): Card-level actions.
+   - A CTA, primary, or solid-filled button labelled with the main call-to-action is required. At times, additional supporting actions/links may be appropriate.
+1. **Description** (optional): A short paragraph that elaborates on the card's purpose.
+1. **Image/Media** (optional): An eye-catching, [brand-approved image](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#p1-photography-style-1) that grabs the user's attention. Use of this area should be reserved for highly promotional cards like events and feature announcements.
+1. **Background** (optional): A background color or [brand-approved texture/gradient](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#c3-color-and-texture-style-3).
+   - When using a texture or gradient, be mindful to maintain appropriate color contrast between the card's text and the background.
+
+#### Presentation
+
+- Call-to-action cards may be presented in a group with other call-to-action cards but also may be presented on their own.
+- When presented in a group, their layout/treatment can be independent from the layout/treatment of the surrounding call-to-action cards.
+
+- Cards that are highly promotional should use elements that create higher visual emphasis (colorful brand-approved images, background colors/textures, etc.).
+  - Use cases (ordered from highest visual emphasis to lowest) include event promotion, new feature release, trial promotion.
+- Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The [white brand-approved textures](https://brandcentral.hpe.com/brand-central/content/distinctive-brand-assets/flexible-ingredients#c3-color-and-texture-style-3) are appropriate if a subtle visual emphasis is desired.
+  - A use case is a software upgrade announcement.
+
+#### Examples
+
+These are example layouts for various use cases. Call-to-action cards that are highly promotional, such as event promotions, incorporate stronger visual elements while those that are less promotional, like software updates, are more subtle.
+
+##### Event promotion
+
+Event announcements are highly promotional, so they should leverage attention-grabbing card elements such as images, that accentuate the dynamic and vibrant experience that an HPE event provides.
 
 <Example
   componentName="Card"
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/CardExample.js"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/EventPromotionCard.js"
```

Comment:
The entire card appears to be clickable in addition to having a button within the card. I think we should have one or the other. Either the entire card is clickable or there is a button not both.

Reply:
We are going to be testing this with Bill tomorrow. The way the example is implemented is the Button itself has `tabIndex={-1}` so there's only a single interactive element. Motivation to explore this was to allow for a larger click target on a card with a single action while still maintaining the "visual" of a button which is a familiar design pattern.


Based on the results of the test, we will build the card in a way that maximizes experience for sighted and non-sighted users.

Here's a link to the test script: https://hpe-my.sharepoint.com/:w:/r/personal/vicky_avalos_hpe_com/_layouts/15/Doc.aspx?sourcedoc=%7BC727E2CB-14DC-4E30-BD83-354709C2BFB3%7D&file=Card%20Testing%20Doc%20with%20Bill.docx&nav=eyJjIjo4NTU3MTIyfQ&action=default&mobileredirect=true&cid=0f0992b1-f079-4d30-a9e7-87388709d437

Reply:
Ah okay that makes sense!